### PR TITLE
Added zmq documentation

### PR DESCRIPTION
**Pull request by hkumarmk**

### CODE REVIEW

File: openstack_zmq.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Why Zmq
+Zmq has brokerless messaging architecture, where individual services talk to
+each other directly in peer-to-peer model.
+
+This brokerless architecture benifit by
+
+1. No central broker, so no single point of failures
+2. Reduced hope as each service directly to other
```
Comment (bodepd) [inline openstack_zmq.md]: can you rephrase this? It does not make sense as-is.

Comment (hkumarmk) [inline openstack_zmq.md]: Ah sure :)
